### PR TITLE
Internal api tests

### PR DESCRIPTION
**Pull request by snayrouz**
#

Description of Changes Made: 
Added active model serializers gem---don't forget to bundle
Anticipated Code Effects: 
(Would this merge affect any other piece of our code?)
Don't merge yet. I keep getting this error 
`Failure/Error: raise ActionController::RoutingError.new('Not Found') unless permission.authorized?`
which makes me think there's something around authorization and this api I need to fix.

I'm unsure if store slugs get passed in the params

I'm also unsure if we want all these actions for these controllers. It doesn't make sense to me that a platform admin can mess with a store item. I'll ask Josh.

DO NOT MERGE THIS YET
[8:32] 
Reviewer:

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -23,6 +23,7 @@ gem 'omniauth-google-oauth2'
 gem 'chart-js-rails'
 gem 'twilio-ruby'
 gem 'figaro'
+gem 'active_model_serializers'
```
Comment (sidewinder2020) [inline Gemfile:26]: excellent

File: spec/requests/api/v1/items_request_spec.rb [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  end
+
+  it 'can delete an item' do
+    item = create(:item)
+
+    expect(Item.count).to eq(1)
+
+    delete "/api/v1/items/#{item.id}"
+
+    expect(response).to have_http_status(:no_content)
```
Comment (sidewinder2020) [inline spec/requests/api/v1/items_request_spec.rb:77]: oooooh nice! - wasn't aware you could set it to :no_content

**Review by sidewinder2020 — COMMENTED** [review]
Looks good - we are a go on JSON output and serializers

**Comment by Aram-Anderson** [comment]
Killed PR. Pulled this branch down locally and merged into the other two PRs that were up.

**Pull request closed without merging**